src/Module.php

```diff
@@ -107,12 +137,34 @@ public function createSignature(SetaPDF_Core_Reader_FilePath $tmpPath)
         $padesDigest = $padesModule->getDigest();
 
         $hashData = hash($padesDigest, $padesModule->getDataToSign($tmpPath), true);
-        $signatureValue = $this->client->signWithOtp(
-            $this->certificateId,
-            $this->signingAlgorithm,
-            $hashData,
-            $this->otp
-        );
+
+        if ( $this->signingMethod == 'sad' ) {
```

Comment:
Please use "===" instead of "==" and remove the spaces around the comparison:

`if ($this->signingMethod === 'sad') {`